server/src/main/java/org/opensearch/extensions/action/IssueServiceAccountResponse.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions.action;
+
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.transport.TransportResponse;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * This class represents a Transport Request for issuing a service account to an extension.
+ */
+public class IssueServiceAccountResponse extends TransportResponse {
+
+    private String name;
+    private String serviceAccountToken;
```

Comment:
Why does the response need to be anything more than an acknowledgement?  The extension either is setup with the account or it doesn't need it / consider it.

Reply:
It doesn't, I just put this information for verifiability etc. Felt it could be good for audit logging. Want me to remove it?

Reply:
Removed so all set.